modules/gdscript/gdscript_function.h

```diff
@@ -342,6 +342,9 @@ class GDScriptFunction {
 		OPCODE_ASSERT,
 		OPCODE_BREAKPOINT,
 		OPCODE_LINE,
+		OPCODE_ITERATE_BEGIN_PACKED_VECTOR4_ARRAY,
+		OPCODE_ITERATE_PACKED_VECTOR4_ARRAY,
+		OPCODE_TYPE_ADJUST_PACKED_VECTOR4_ARRAY,
```

Comment:
It probably makes more sense to add the opcodes next to similar ones, not to the end. This does not break compatibility and does not require additional actions such as increasing the bytecode version (`.gdc` is not bytecode).

Reply:
I thought we wanted to put entries at the end so the numerical values stay the same. Is this not correct?

Reply:
No, see [git blame](https://github.com/godotengine/godot/blame/f15ced3f188b2dfa3f2b9f1b1cda56774977b24f/modules/gdscript/gdscript_function.h#L199).